For Advanced/010.md

```diff
@@ -0,0 +1,17 @@
+# Vital Selenium
+
+Now, since you know the basics, it’s time to leverage some important and unique features of G1ANT.Language. Let’s try the commands that master the Internet — starting with the ones, which communicate with browsers.
+
+The `selenium.click` command is used to click an element on a webpage that was open using the `selenium.open` command. Both commands are provided by the Selenium Addon, so in order to use them, you need to enable the addon in the Addons panel to the left of the script editor: just click the check box next to the **selenium** entry to select it.
+
+Run this script:
+
+```G1ANT
+selenium.open chrome url google.com
+window ✱Google✱ style maximize
+selenium.click search gb_70 by id
```

Comment:
I guess there might be some time needed for the page to load.

Reply:
if it opens fast enough on my computer, it will be OK everywhere else ;)